packages/rpms/SPECS/wazuh-agent.spec

```diff
@@ -17,7 +17,7 @@ Group:       System Environment/Daemons
 Source0:     %{name}-%{version}.tar.gz
 URL:         https://www.wazuh.com/
 BuildRoot:   %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
-Vendor:      Wazuh, Inc <info@wazuh.com>
+Vendor:      Wazuh <info@wazuh.com>
 Packager:    Wazuh, Inc <info@wazuh.com>
```

Comment:
Should we also remove "Inc" from the `Packager` field?

Reply:
Changed Packager field to match with vendor.
Check that this change not affect Wazuh-agent package:
- https://github.com/wazuh/wazuh-agent-packages/actions/runs/16501314154
```
[root@fedora41 wazuh]# sqlite3 /var/ossec/queue/db/006.db "SELECT DISTINCT vendor FROM sys_programs;"
Fedora Project
Wazuh
```